code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2423,11 +2424,14 @@
 	if((prob(10) && method == TOUCH) || method == INGEST)
 		M.contract_disease(new /datum/disease/robotic_transformation(0), 1)
```

Comment:
Just make this disease typepath a variable that differs between the two, and delete the reaction mob proc on the mommi version